indexer/src/ReleaseSyncer.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+import { Release } from '@nina-protocol/nina-db';
+import { logTimestampedMessage } from './utils/logging.js';
+
+class ReleaseSyncer {
```

Comment:
@facing-n I think we should rename this to something like RestrictedReleaseFilter or Deleter or something like that, what do you think?